cmd/controller/checknodehealth/main.go

```diff
@@ -46,11 +48,29 @@ func main() {
 	flag.Parse()
 	defer klog.Flush()
 
+	// Set up controller-runtime logger
+	ctrl.SetLogger(zap.New(zap.UseDevMode(true)))
```

Comment:
We should also maybe also call `klog.SetLogger` so that we don't end up with multiple log formats.